util/build_configs/setenv-gh9943.bash

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
Is there a more descriptive name we could give this file? I don't think 'gh9943' will be useful to most users.

Reply:
Arg- I meant to have a README in this directory, but forgot. 
I shall do that later today. While at it, maybe some better file names for example setenv scripts will occur to me. Any suggestions? Thanks again.